plugins/modules/identity/ipa/ipa_user.py

```diff
@@ -146,11 +147,11 @@
     ipa_pass: topsecret
     update_password: on_create
 
-- name: Ensure pinky is present and using one time password authentication
+- name: Ensure pinky is present and using one time password and RADIUS authentication
   community.general.ipa_user:
     name: pinky
     state: present
-    userauthtype: otp
+    userauthtype: ['otp', 'radius']
```

Comment:
I think it's easier to read this way:
```suggestion
    userauthtype:
      - otp
      - radius
```

Reply:
> I'm not sure how idempotency checking works for the module, and whether this change is backwards compatible with respect to idempotency checking. Do you know, and/or did you check?

I did.  I first used the original by putting a 'password' option in the str field and executed it twice, confirming the first time was 'changed' and the second time was 'ok'.  Then I made the patch, and asked for the same 'password' setting (not in a list form) and it was 'ok' (not changed).  I subsequently moved to 'otp' and 'radius' (twice) and confirmed correct behavior and back to just 'password'.  Looked like it was working correctly, at least in my setup.

Reply:
@justchris1 thanks for verifying this!